Contest_API.md

```diff
@@ -594,8 +594,8 @@ Note that all results returned from endpoints:
 Endpoints that return a JSON array must allow filtering on any
 property with type ID (except the `id` property) by passing it as a
 query argument. For example, clarifications can be filtered on the
-recipient by passing `to_team_id=X`. To filter on a `null` value,
-pass an empty string, i.e. `to_team_id=`. It must be possible to
+recipient by passing `to_team_ids=X,Y`. To filter on a `null` value,
+pass an empty string, i.e. `to_team_ids=`. It must be possible to
```

Comment:
This suggests a real change: instead of just requiring filtering on properies with `ID` type, we would now also require filtering on properties with type `array of ID`.

That adds a bunch of extra complexity like: if you specify `to_team_ids=X,Y`, do you want to match entries where `to_team_ids` is exactly `[X,Y]`, contains either `X` or `Y`, has `[X,Y]` as a subset, or... I don't think we should go there, and simply change this example to something else than clarifications.

Reply:
The "OR" of all the to_team_ids  (or to_group_ds) makes the most sense.  eg.  "Any of these teams"

Reply:
> I don't think we should go there, and simply change this example to something else than clarifications.

I agree, we should not add the requirement that you allow filtering on any property of type `array of ID`.

On a related note, when we say "any property with type `ID`" do we mean to include properties of type `ID ?`. We should clarify.